src/main/scala/dsptools/DspTester.scala

```diff
@@ -247,12 +247,11 @@ class DspTester[T <: Module](c: T,
     }
   }
 
-  def dspExpect(data: DspComplex[_], expected: Complex, msg: String): Unit = {
+  def dspExpect(data: DspComplex[_], expected: Complex, msg: String): Boolean = {
     dspPeek(data) match {
       case Right(complex) =>
         println(f"expect got $complex expect $expected")
-        expect(nearlyEqual(complex.real, expected.real), msg)
-        expect(nearlyEqual(complex.imag, expected.imag), msg)
+        expect(nearlyEqual(complex.real, expected.real), msg) & expect(nearlyEqual(complex.imag, expected.imag), msg)
```

Comment:
It shouldn't be. & is bitwise-and, && is logical-and. You're returning a value that represents an outcome, right? true->pass, false->fail? That should be logical-and. For boolean it works out to being the same, but it's just a little confusing b/c with & you always have to go check the types of things you're and-ing.